Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/settings/ChanState.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.adamantcheese.chan.core.settings
+
+import com.github.adamantcheese.chan.utils.AndroidUtils
+import com.github.adamantcheese.chan.utils.Logger
+
+
+object ChanState {
```

Comment:
Actually this class is still unmockable. I need to create an interface with all the settings methods and inherit this singleton from the interface. And with that I will be able to create mocks of the interface.